vscode-dotnet-runtime-library/distro-data/distro-support.json

```diff
@@ -1,17 +1,90 @@
 {
 	"$schema": "../../Documentation/json-schema/distro-support-schema.json",
 	"Ubuntu": {
-		"installCommand": "sudo apt-get update && sudo apt-get install -y {0}",
-		"uninstallCommand": "sudo apt-get remove {0}",
-		"updateCommand": "sudo apt-get update && apt-get upgrade -y {0}",
-		"isInstalledCommand": "sudo apt list --installed {0}",
-		"packageLookupCommand": "dpkg -l {0}",
+		"installCommand":
```

Comment:
the schema file needs to be updated for this change, I believe.

Reply:
Indeed, I will need to rewrite it again for when I add the searchCommand, which I am doing now! :)